schema/formats_test.go

```diff
@@ -90,6 +90,83 @@ var _ = Describe("format checkers", func() {
 			result := formatChecker.IsFormat("::1/129")
 			Expect(result).To(Equal(false))
 		})
+
+		It("Should not pass - two zeros in octet IPv4", func() {
+			result := formatChecker.IsFormat("127.00.0.1/24")
+			Expect(result).To(Equal(false))
+		})
+
+		It("Should not pass - three zeros in octet IPv4", func() {
+			result := formatChecker.IsFormat("127.000.0.1/24")
+			Expect(result).To(Equal(false))
+		})
+
+		It("Should not pass - two zeros in first octet IPv4", func() {
+			result := formatChecker.IsFormat("000.0.0.1/24")
```

Comment:
test data does not match test name (should be "**00**.0...")

Reply:
done